src/main/scala/com/localytics/sbt/sqs/ElasticMQKeys.scala

```diff
@@ -0,0 +1,24 @@
+package com.localytics.sbt.sqs
+
+import java.io.File
+import sbt._
+
+object ElasticMQKeys {
+
+  case class QueueConf(name: String, visibilityTimeoutSecs: Int = 10, delaySecs: Int = 5, receiveMessageWaitSecs: Int = 0)
+  case class NodeAddressConf(protocol: String = "http", host: String = "localhost", port: Int = 9324, contextPath: String = "")
+  case class RestSQSConf(enabled: Boolean = true, bindPort: Int = 9324, bindHostname: String = "0.0.0.0", sqsLimits: String = "strict")
+
+  lazy val elasticMQDir        = settingKey[File]("The directory ElasticMQ will be downloaded to. Defaults to 'elastic-mq'")
+  lazy val elasticMQUrl        = settingKey[String]("The URL to download ElasticMQ from. Defaults to 'https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-0.8.12.jar'")
```

Comment:
I would consider adding `{version}` to the url here so we don't have to update it when we bump ElasticMQ versions.